tests/test_backend.py

```diff
@@ -51,11 +51,6 @@
 
 # Ops
 # Missing ops
-# Missing op 'ConvTranspose' -> NGRAPH-1840
-backend_test.exclude('test_ConvTranspose2d_cpu')
-backend_test.exclude('test_ConvTranspose2d_no_bias_cpu')
-backend_test.exclude('test_operator_convtranspose_cpu')
-
```

Comment:
Add some our UT, you may use examples from [doc](https://github.com/onnx/onnx/blob/master/docs/Operators.md#examples-16)

Reply:
I don't think we need to add custom tests if there are ONNX tests covering `ConvTranspose`.